test/parallel/test-zlib-from-gzip-with-trailing-garbage.js

```diff
@@ -28,9 +32,12 @@ data = Buffer.concat([
   Buffer(10).fill(0)
 ]);
 
-assert.throws(() => zlib.gunzipSync(data));
+assert.throws(
+  () => zlib.gunzipSync(data),
+  /^Error: unknown compression method$/
+);
 
-zlib.gunzip(data, common.mustCall((err, result) => {
+zlib.gunzip(data, common.mustCall((err) => {
   assert(err);
```

Comment:
New commit pushed up with these suggestions included. Thanks to the both of you for the help. Long time listener, first time committer. 😄